good-enough-practices-for-scientific-computing.tex

```diff
@@ -281,42 +230,29 @@ \section{Data Management}\label{sec:data}
   \withurl{Dryad}{http://datadryad.org/}, and
   \withurl{Zenodo}{https://zenodo.org/} allow others to find your work,
   use it, and cite it; we discuss licensing in
-  Section~\ref{sec:collaboration} below.
+  Section~\ref{sec:collaboration} below. Metadata should also be stored 
+  
+  \fixme{anyone}{Let's modify the preceding item about deposition to include something about
```

Comment:
I removed a long paragraph about metadata. But think we can probably modify the existing item about deposition in a repository to include both this sentiment and the original Wickes citation. This is a TO DO.